modules/virt-configuring-secondary-dns-server.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-accessing-vm-secondary-network-fqdn.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-secondary-dns-server_{context}"]
+= Configuring DNS server for secondary networks
+
+The Cluster Network Addons Operator (CNAO) deploys the Domain Name Server (DNS) server and monitoring components when you enable the `KubeSecondaryDNS` feature gate in the `HyperConverged` custom resource (CR).
+
+.Prerequisites
+* You installed the OpenShift CLI (`oc`).
+* You have access to an {product-title} cluster with `cluster-admin` permissions.
+
+.Procedure
+. Create a `LoadBalancer` service using MetalLB or any other load balancer to expose the DNS server outside the cluster. The service listens on port 53 and targets port 5353. For example:
++
+[source,terminal]
+----
+$ oc expose -n openshift-cnv deployment/secondary-dns --name=dns-lb --type=LoadBalancer --port=53 --target-port=5353 --protocol='UDP'
+----
+
+. Retrieve the public IP address of the service by querying the `Service` object:
```

Comment:
I imagine actual users will know which IP is the "public" IP. I would have to guess that it's the external. 

Don't  know if you should s/public/external

Reply:
AFAIK, the terms are interchangeable.